### ISSUE TITLE
Status mail wrong encoding

### ISSUE THREAD

**Issue by DrCarsonBeckett** [webmin/usermin#67]
We're about to release Webmin 1.962 and Usermin 1.811, which are going to include these patches. Within few days, as you install updates, please come back to us sharing how it works for you.

_Originally posted by @iliarostovtsev in https://github.com/webmin/webmin/issues/1272#issuecomment-725585499_

I need to open another ticket for that too.
I received a few mails and the issue is still there.
It happens in different browsers (IE, Edge, Firefox, iOS Safari and iOS Firefox) as well on other PCs. It happens in Outlook and Thunderbird as well, the only mail app which displays it correctly for me is the iOS default Mail app.

System:
Debian 10
Webmin-Version | 1.962
Usermin-Version | 1.812
Virtualmin Version | 6.14
Authentic theme version | 19.62

Edit: I opened the issue first in the webmin area and closed it (#1332), sorry.

**Comment by iliarostovtsev** [comment]
Hi,

How this issue is different from https://github.com/webmin/usermin/issues/66?

> It happens in different browsers (IE, Edge, Firefox, iOS Safari and iOS Firefox) as well on other PCs. It happens in Outlook and Thunderbird as well

What happens?

> I opened the issue first in the webmin area and closed it (#1332), sorry.

No problem, thanks!

**Comment by DrCarsonBeckett** [comment]
I am not sure if its related to that or if its something else. This is about the encoding in the mails and not on the website.

> What happens?
That the characters in the mail are getting displayed wrong:
> Backup ist komplett. Die GesamtgrÃ¶ÃŸe betrÃ¤gt 

Which should be displayed as:
> Backup ist komplett. Die Gesamtgröße beträgt

**Comment by iliarostovtsev** [comment]
> I am not sure if its related to that or if its something else. This is about the encoding in the mails and not on the website.

Considering it's generated by Virtualmin, it must be our bug, right @jcameron :slightly_smiling_face:

**Comment by jcameron** [comment]
Looks like our bug. Can you post the email here as an attachment, with all headers?

**Comment by DrCarsonBeckett** [comment]
I prefer not to do that due to the fact that data would be public then and I prefer that some of them are not.
I can send them however via pm if that's somewhat possible.

**Comment by iliarostovtsev** [comment]
Yes, you can send it to me using my website's form. We will deal with it.

**Comment by DrCarsonBeckett** [comment]
I just sent it.

**Comment by iliarostovtsev** [comment]
This https://github.com/webmin/webmin/commit/1f9e4795caec8b332678beca2e25b809cddd41b2 should fix this. Could you test it please?

**Comment by DrCarsonBeckett** [comment]
Tested it, its still the same.
From what I read in the patch / note in the commit, its no wonder since the message(-content) itself is displayed wrong and not only the subject. Or did I misunderstood that?

**Comment by DrCarsonBeckett** [comment]
The odd thing is, that only the backup report mail seems to be wrong. All other emails are encoded correctly (everywhere).
Even usermin got issues displaying it correctly, so its not only related to Outlook.

**Comment by jcameron** [comment]
I looked into this, and it turns out that the code for sending backup status emails in Virtualmin is different and needed to be fixed separately.

**Comment by DrCarsonBeckett** [comment]
From what I talked with @iliarostovtsev , said the same. If I understood correctly, you guys added encoding to it. And he said, it should be fixed with the latest virtualmin package (6.14). So far, from the tests we did, it looks good ... at least for me.
The last status mail I recieved was "bug" free on all apps and devices. 😃

**Comment by iliaross** [comment]
❗️ If the issue isn't resolved, feel free to reopen it with updated details.
